demisto_sdk/commands/create_artifacts/content_creator.py

```diff
@@ -133,6 +139,37 @@ def add_from_version_to_json(file_path):
 
         return json_content
 
+    def add_suffix_to_file_path(self, file_path):
+        return os.path.splitext(file_path)[0] + self.file_name_suffix + os.path.splitext(file_path)[1]
+
+    def check_from_version(self, file_path):
```

Comment:
function name doesn't explain its purpose very well...
check for what?

Reply:
done